Dot 3 Github.py

```diff
@@ -550,14 +552,19 @@ async def setqueue(interaction: nextcord.Interaction, queue_type: str = nextcord
     await interaction.response.send_message(f"The queue for {queue_type} has been updated with the mentioned players.")
 
 async def start_user(interaction, user):
-    add_active_storyteller(user, queue[str(user.id)]["QueueType"])
-    await remove_queue(user.id)
-    await interaction.response.send_message(f"{user.display_name} is now active.", ephemeral=False)
-    #await check_queue()
-
-    # Notify user after 40 minutes
-    await asyncio.sleep(RE_RACK_TIMER_DURATION)
-    await interaction.channel.send(f"{user.mention}, the Re-rack timer has expired.")
+    if str(user.id) in queue:

```

Comment:
Similar comment as in the other pr:
```suggestion
    if str(user.id) not in queue:
        await interaction.response.send_message("The ST you are trying to start is not in queue and therefore can't be started.")
        return
    add_active_storyteller(user, queue[str(user.id)]["QueueType"])

```